users/syncthing: Document the generate subcommand and its options (#694)
syncthing/docs: users/syncthing.rst

signal handler. Exit codes over 128+N on Unix usually represent the signal which
caused the process to exit. For example, ``128 + 9 (SIGKILL) = 137``.

Sub-Commands
------------

The command line syntax actually supports different modes of operation through
several sub-commands, specified as the first argument.  The default one is

> ``serve`` and can therefore be omitted.

Reviewer: That's just for backwards compatibility, I don't think it should be documented and thus it's use encoured.

Author: There are tons of examples out there in the Internets which don't include `serve`, where the documentation would not explain why they work at all.

```suggestion
several sub-commands, specified as the first argument.  The default one is
``serve``, assumed when none is given.
```

No longer a suggestion to do so, but still explains what will happen.  Will commit a better wording...
